www/js/diary/diaryHelper.ts

```diff
@@ -114,11 +114,15 @@ export function getFormattedDate(beginFmtTime: string, endFmtTime?: string) {
     return `${getFormattedDate(beginFmtTime)} - ${getFormattedDate(endFmtTime)}`;
   }
   // only one day given, or both are the same day
-  const t = moment.parseZone(beginFmtTime || endFmtTime);
-  // We use ddd LL to get Wed, May 3, 2023 or equivalent
-  // LL only has the date, month and year
-  // LLLL has the day of the week, but also the time
-  return t.format('ddd LL');
+  const t = DateTime.fromISO(beginFmtTime || endFmtTime);
```

Comment:
Please ensure that you are parsing ***with respect to timezone***
https://github.com/moment/luxon/issues/221#issuecomment-519194393

If not, we could end up on the wrong day sometimes.